### PR TITLE
Addresses #1814 [NTR] early and late spermatid

### PR DESCRIPTION
**Pull request by bvarner-ebi**
Addresses #1814 [NTR] early and late spermatid

### CODE REVIEW

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -30617,6 +30619,22 @@ AnnotationAssertion(rdfs:label obo:CL_4030035 "dental pulp stem cell")
...
 SubClassOf(obo:CL_4030035 obo:CL_0002148)
 
+# Class: obo:CL_4030036 (early spermatid)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "https://www.proteinatlas.org/humanproteome/single+cell+type/Germ+cells#earlyspermatids"^^xsd:string) obo:IAO_0000115 obo:CL_4030036 "A spermatid in an early stage of maturation that has a round morphology and is transcriptionally active."^^xsd:string)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4030036 "2023-02-01T13:56:00Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:CL_4030036 "early spermatid")
+SubClassOf(obo:CL_4030036 obo:CL_0000018)
+SubClassOf(obo:CL_4030036 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0000079))
+
```
Comment (aleixpuigb) [inline src/ontology/cl-edit.owl:30629]: Consider adding 'capable of' some 'spermatid development' (GO:0007286) and DbXref: https://doi.org/10.1016/B978-0-323-01970-5.50014-2

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -30617,6 +30619,22 @@ AnnotationAssertion(rdfs:label obo:CL_4030035 "dental pulp stem cell")
...
+# Class: obo:CL_4030037 (late spermatid)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "https://www.proteinatlas.org/humanproteome/single+cell+type/Germ+cells#latespermatids"^^xsd:string) obo:IAO_0000115 obo:CL_4030037 "A spermatid in a late stage of maturation that has an elongated morphology and is transcriptionally inert when the acrosome is fully developed."^^xsd:string)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4030037 "2023-02-01T14:02:15Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:CL_4030037 "late spermatid")
+SubClassOf(obo:CL_4030037 obo:CL_0000018)
+SubClassOf(obo:CL_4030037 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0000079))
+
```
Comment (aleixpuigb) [inline src/ontology/cl-edit.owl:30637]: Consider adding 'capable of' some 'spermatid development' (GO:0007286) and DbXref: https://doi.org/10.1016/B978-0-323-01970-5.50014-2
also consider adding 'develops from' some 'early spermatid'

File: src/ontology/cl-edit.owl (hunk 1 above)
Comment (aleixpuigb) [reply src/ontology/cl-edit.owl:30629]: In the same reference (and others), 'round spermatid' and 'early spermatid' are used as synonyms.

File: src/ontology/cl-edit.owl (hunk 2 above)
Comment (aleixpuigb) [reply src/ontology/cl-edit.owl:30637]: In the same reference (and others), 'elongated spermatid' and 'early spermatid' are used as synonyms.

**Review by aleixpuigb — APPROVED** [review]

File: src/ontology/cl-edit.owl (hunk 1 above)
Comment (bvarner-ebi) [reply src/ontology/cl-edit.owl:30629]: Thank you for the suggestions. I have incorporated these suggestions in the latest commit.

File: src/ontology/cl-edit.owl (hunk 2 above)
Comment (bvarner-ebi) [reply src/ontology/cl-edit.owl:30637]: Thank you for the suggestions. I have incorporated these suggestions in the latest commit.

**Review by aleixpuigb — APPROVED** [review]

**Pull request merged**